### ISSUE TITLE
Patent US6252960 expired, okay to implement compressed curve points?

### ISSUE THREAD

**Issue by joernheissler** [wbond/asn1crypto#161]
https://github.com/wbond/asn1crypto/blob/master/asn1crypto/keys.py#L191

According to https://patents.google.com/patent/US6252960B1/en, the patent expired. I'm not sure if that was in Aug 2018 or in Oct 2019.

I'm by no means an expert on patents, so I could be mistaken here. Do you think this feature can be implemented now?

**Comment by wbond** [comment]
Yes, the last time I looked (sometime back in October), it seemed pretty much all of the EC patents had finally expired.

I'm fine with implementing it now. I don't think I'll have time in the near future, but would certainly be fine with reviewing it if someone has time to write up a PR with some real-world test cases.
